--- a/arch/arm/dts/org/Makefile	2025-07-08 00:48:28.000000000 +0900
+++ b/arch/arm/dts/Makefile	2025-11-12 01:08:58.667393873 +0900
@@ -650,7 +650,9 @@
 	sun8i-s3-elimo-initium.dtb \
 	sun8i-s3-pinecube.dtb \
 	sun8i-v3-sl631-imx179.dtb \
-	sun8i-v3s-licheepi-zero.dtb
+	sun8i-v3s-licheepi-zero.dtb \
+	sun8i-v3s-licheepi-zero-dock.dtb \
+	sun8i-v3s-licheepi-zero-dock-dx.dtb
 dtb-$(CONFIG_MACH_SUN8I_R528) += \
 	sun8i-t113s-mangopi-mq-r-t113.dtb
 dtb-$(CONFIG_MACH_SUN50I_H5) += \
